src/report.rs

```diff
@@ -404,6 +406,46 @@ pub fn error_row(error: &metrics::GooseErrorMetricAggregate) -> String {
     )
 }
 
+pub fn graph_rps_template(rps: Vec<f32>, bucket_size: usize) -> String {
+    let values = json!(rps);
+    let buckets = json!(
+        (0..rps.len()).
+            map(|bucket| format!("{}s - {}s", (bucket * bucket_size), ((bucket + 1) * bucket_size)))
+            .collect::<Vec<_>>()
+    );
+
+    format!(
+        r#"<div class="graph-rps">
+        <h2>Requests per second</h2>
+            <div id="main" style="width: 1000px; height:660px; background: white;"></div>
+
+            <script src="https://cdn.jsdelivr.net/npm/echarts@5.2.2/dist/echarts.min.js"></script>
```

Comment:
Is it OK to load echarts from the CDN?Is it OK to load echarts from the CDN?

Reply:
That's fine to start - we could make this configurable in the future - if you wanted to host your own.

Reply:
Great points. Mentioned the library in the docs.